nimare/meta/cbma/ale.py

```diff
@@ -176,7 +189,12 @@ def _determine_histogram_bins(self, ma_maps):
         # Assuming values of 0, .001, .002, etc., bins are -.0005-.0005, .0005-.0015, etc.
         INV_STEP_SIZE = 100000
         step_size = 1 / INV_STEP_SIZE
-        max_ma_values = np.max(ma_values, axis=1)
+        if isinstance(ma_values, sparse._coo.core.COO):
+            # Need to convert to dense because np.ceil is too slow with sparse
+            max_ma_values = ma_values.max(axis=[1, 2, 3]).todense()
+        else:
+            max_ma_values = np.max(ma_values, axis=1)
```

Comment:
Can you add a comment with info about _when_ this would be reached? I assume it would be reached when `ma_maps` is a list above, but if there are other cases then I'd like those documented as well.

Reply:
I'm deleting these lines since `_collect_ma_maps` always returns a sparse array now.